pkg/gen/input/workflows/internal/file/create_release_pr.go

```diff
@@ -62,26 +62,23 @@ jobs:
           echo "::set-output name=base::${base}"
           echo "::set-output name=head::${head}"
           echo "::set-output name=version::${version}"
-          echo "::endgroup::"
-          echo "::group::Validate"
-          git init -q
-          git remote add origin ${{ github.event.repository.clone_url }}
-          git fetch -q --depth=1 origin $base
-          git fetch -q --depth=1 origin $head
-          out=$(git rev-list --left-right --count origin/$base...origin/$head)
-          ahead=$(echo $out | awk '{print $2}')
-          behind=$(echo $out | awk '{print $1}')
-          echo "ahead=\"$ahead\" behind=\"$behind\""
-          if [[ $ahead -ne 0 ]] || [[ $behind -ne 0 ]] ; then
-            echo "::error:: Branch $head is $ahead commits ahead and $behind commits behind $base branch. Please recreate the $head branch."
-            exit 1
+      - name: Check if PR exists
+        id: pr_exists
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+        run: |
+          if gh pr view --repo ${{ github.repository }} ${{ github.event.ref }} ; then
+            echo "::warning::Release PR already exists"
+            echo "::set-output name=skip::true"
+          else
+            echo "::set-output name=skip::false"
           fi
-          echo "::endgroup::"
   create_release_pr:
     name: Create release PR
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
     needs:
       - gather_facts
+    if: ${{ needs.gather_facts.outputs.skip != 'true' }}
```

Comment:
... and if skip istrue, don't run this step:

<img width="854" alt="Screenshot 2020-10-15 at 18 07 55" src="https://user-images.githubusercontent.com/4608994/96163169-73682d80-0f11-11eb-9fd0-e9aa33e61b2e.png">